decidim-conferences/app/views/decidim/conferences/conferences/_promoted_conference.html.erb

```diff
@@ -1,9 +1 @@
-<%= link_to conference_path(promoted_conference), id: "promoted-#{dom_id(promoted_conference)}", class: "card__highlight" do %>
-  <div class="card__highlight-img">
-    <%= image_tag promoted_conference.attached_uploader(:hero_image).path, alt: t(promoted_conference.cta_button_text_key, scope: "layouts.decidim.conferences.promoted_conference") %>
-  </div>
-  <div class="card__highlight-text">
-    <h3 class="h3 text-secondary"><%= translated_attribute promoted_conference.title %></h3>
-    <%= decidim_sanitize_editor html_truncate(translated_attribute(promoted_conference.short_description), length: 630, separator: "...") %>
-  </div>
-<% end %>
+<%= card_for promoted_conference, highlight: true %>
```

Comment:
Why do you still use the partial to render the card? 

The only place i could easily find is: ```decidim-conferences/app/views/decidim/conferences/conferences/index.html.erb:23```

Maybe you could call there the Card ?

Reply:
Moved to the view directly